journey.txt

```diff
@@ -16,4 +16,9 @@
 (3/16/2020)
   -made changes to database to include a rating property, current version, and # of ratings and fixed tests.
   -used a css flex box layout to structure my data in react component. things got really messy, so might try a different approach.
-  -split my info component from my main index.jsx component
\ No newline at end of file
+  -split my info component from my main index.jsx component
+(3/17/2020-3/22/2020)
+  -switched from flexbox layout to css grid layout which made it alot easier to organize all of my data points. (this process took a long time)
```

Comment:
I'll have to try grid next time.  I used bootstrap for mine which is a derivative of flexbox and I feel like its been unnecessarily difficult to work with.